dht_bootstrap.go

```diff
@@ -119,6 +120,20 @@ func (dht *IpfsDHT) startRefreshing() error {
 			if err != nil {
 				logger.Warning(err)
 			}
+
+			// ping Routing Table peers that haven't been hear of/from in the interval they should have been.
+			for _, ps := range dht.routingTable.GetPeerInfos() {
+				// ping the peer if it's due for a ping and evict it if the ping fails
+				if time.Since(ps.LastSuccessfulOutboundQuery) > dht.maxLastSuccessfulOutboundThreshold {
```

Comment:
We're not treating this as a successful query. I think we need to record a _separate_ "last seen"? Otherwise, we'll keep re-connecting once the peer hits this limit.

Reply:
As discussed offline, this is not needed for now as we will probably evict flaky-connectivity peers from the DHT because they wont respond back to us with good latencies. Please see https://github.com/libp2p/go-libp2p-kad-dht/pull/536 for more details.